lib/humanize/words.rb

```diff
@@ -27,6 +33,8 @@ module Humanize
     },
     :tr => {
       :group_by => 1_000,
+      :infinity => 'sonsuzluk', # needs verification
```

Comment:
Please use 'sonsuz'. The other one is ok. Thanks.

Reply:
Thank you @mertbulan.